cmd/eudico/mirvalidator/run.go

```diff
@@ -60,6 +61,16 @@ var runCmd = &cli.Command{
 			Name:  "init-checkpoint",
 			Usage: "pass initial checkpoint as a file (it overwrites 'init-height' flag)",
 		},
+		&cli.StringFlag{
```

Comment:
We should pass a flag to point to the ipc-agent endpoint and use this throughout the code when requiring an interaction with the agent. We should do the same also for the lotus daemon.

Reply:
Should we address that in a separate PR?

Reply:
Sounds good, feel free to open an issue to track it and let's address it next. Thanks.